diff --git a/account_financial_report/__manifest__.py b/account_financial_report/__manifest__.py
index 62a6c3c..5f7323a 100644
--- a/account_financial_report/__manifest__.py
+++ b/account_financial_report/__manifest__.py
@@ -15,7 +15,7 @@
     "ForgeFlow,"
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-financial-reporting",
-    "depends": ["account", "date_range", "report_xlsx"],
+    "depends": ["account", "date_range", "report_xlsx", "usl_accounting"],
     "data": [
         "security/ir.model.access.csv",
         "security/security.xml",
