diff --git a/account_reconcile_oca/static/src/components/selection_badge_uncheck/selection_badge_uncheck.esm.js b/account_reconcile_oca/static/src/components/selection_badge_uncheck/selection_badge_uncheck.esm.js
index 0c461be..8cbfc12 100644
--- a/account_reconcile_oca/static/src/components/selection_badge_uncheck/selection_badge_uncheck.esm.js
+++ b/account_reconcile_oca/static/src/components/selection_badge_uncheck/selection_badge_uncheck.esm.js
@@ -1,27 +1,8 @@
-import {
-    BadgeSelectionField,
-    badgeSelectionField,
-} from "@web/views/fields/badge_selection/badge_selection_field";
+import {badgesMany2oneField} from "@web/views/fields/badges_many2one/badges_many2one_field";
 import {registry} from "@web/core/registry";
 
-export class FieldSelectionBadgeUncheck extends BadgeSelectionField {
-    async onChange(value) {
-        var old_value = this.props.value;
-        if (this.props.type === "many2one") {
-            old_value = old_value[0];
-        }
-        if (value === old_value) {
-            this.props.update(false);
-            return;
-        }
-        super.onChange(...arguments);
-    }
-}
-
 export const FieldSelectionBadgeUncheckField = {
-    ...badgeSelectionField,
-    component: FieldSelectionBadgeUncheck,
-    supportedTypes: ["many2one"],
+    ...badgesMany2oneField,
 };
 registry
     .category("fields")
